platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/MapView.java

```diff
@@ -305,6 +305,7 @@ public void onSurfaceCreated(GL10 gl, EGLConfig config) {
       addView(glSurfaceView, 0);
     }
 
+    mapRenderer.maximumFps(options.getMaximumFps());
```

Comment:
Thanks for doing the `MapboxMapOptions` integration, but I'd still love to see this exposed via `MapView` so that the FPS cap can be altered while the `MapView` is visible, not only on inflation.

Reply:
I missed that it could be usefull changing this at runtime. Feel free to revert

Reply:
I was more thinking about keeping both tbh! We can still keep `MapboxMapOptions` without removing a possibility to update the setting in the runtime.